.gitconfig

```diff
@@ -0,0 +1,2 @@
+user.email = shiraeeshi@mail.ru
+user.name = shiraeeshi
```

Comment:
Usually user want to config it globally
https://help.github.com/articles/set-up-git/#setting-up-git
Or use local config for this repo in .git/config